tests/unit/core/test_water.py

```diff
@@ -80,6 +81,46 @@ def test_calc_viscosity_h20_matrix(shape):
     assert np.allclose(eta.round(7), np.full(shape, fill_value=0.0010016))
 
 
+def test_calculate_water_molar_volume():
+    """Simple sense check that molar volume at standard conditions ~= molar mass."""
+
+    from pyrealm.constants import CoreConst
+    from pyrealm.core.water import calculate_water_molar_volume
+
+    assert_allclose(
+        calculate_water_molar_volume(tc=0, patm=101325),
+        CoreConst.k_water_molmass,
+        atol=0.01,
```

Comment:
Is there a reason you are using atol here and rtol elsewhere?

Reply:
No 😄